<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MilfPy</class>
 <widget class="QMainWindow" name="MilfPy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MilfPy0.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/milfICON.PNG</normaloff>img/milfICON.PNG</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mainBtn1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="mainTB1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>34</y>
      <width>216</width>
      <height>346</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn2">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>0</y>
      <width>60</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>중지</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>305</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>기본</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>287</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>설정</string>
      </property>
      <widget class="QCheckBox" name="checkBox1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>84</y>
         <width>137</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>무한반복</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>34</y>
         <width>56</width>
         <height>33</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>38</y>
         <width>102</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>반복횟수</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>137</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>룬체크 (딥러닝) 안함</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>디버그</string>
     </attribute>
     <widget class="QPushButton" name="mainBtn4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>룬찾기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>81</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>테스트버튼1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>81</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>테스트버튼2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>81</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>테스트버튼3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>셋팅</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="mainBtn3">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>0</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>불러오기</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>378</y>
      <width>1366</width>
      <height>766</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap>../MiraiOfSetsuna/data/img/testImg.jpg</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
